migreringer/src/main/resources/db/postgres/defaultDS/1.0/V1.0_065__enum_ordinal_til_kode.sql

```diff
@@ -0,0 +1,39 @@
+alter table PART add column identifikasjon_type_kode varchar(50);
```

Comment:
Lar den gamle kolonna fortsatt ligge slik at den deployede versjonen av applikasjonen fortsette å lese. Det vil ikke være mulig for den å skrive siden den nye kolonna er not null.